docs/Dockerfile

```diff
@@ -2,9 +2,9 @@
 # This dockerfile was used to generate darthtrevino/dowhy-docs, used in the 
 # build-and-publish-docs.yml workflow
 #
-FROM pego/dowhy-examples-notebooks-deps
+FROM ghcr.io/py-why/dowhy-example-notebooks-deps
```

Comment:
I'm assuming this comes packed with all the R deps? Do we have a plan for deprecating the R notebooks?

Reply:
> I'm assuming this comes packed with all the R deps?

yes

> Do we have a plan for deprecating the R notebooks?

Yes. I've already removed R dependencies in a couple of notebooks that use the Lalonde dataset and use R only to load it (see recent commit history).

Actually, the plan is not to deprecate the notebooks, but to replace the R code in there with native Python. We'll need to take inventory and see what this will mean concretely. I.e., how many notebooks are we actually talking about. And then what algorithms are used and what can already be replaces.